README.md

```diff
@@ -39,7 +39,10 @@ Examples to run on RunII legacy test campaign:
 And a very basic workflow test (for reference, this is not supposed to give paper-grade results):
  ```
  cd $CMSSW_BASE/src/flashgg
- cmsRun MicroAOD/test/microAODstd.py processType=sig datasetName=glugluh # or processType=data depending on input file
+cmsRun MicroAOD/test/microAODstd.py processType=sig year=2017 datasetName=glugluh conditionsJSON=MetaData/data/MetaConditions/Era2016_RR-17Jul2018_v1.json 
```

Comment:
remove year=2017 from the example to match the changes in the code

Reply:
Done.